lessons/Algebra2/exploring-quadratic-models/langs/en-us/pages/graphing-models.adoc

```diff
@@ -0,0 +1,62 @@
+= Graphing Quadratic Models
+++++
+<style>
+.autonum { font-weight: bold; }
+.autonum:after { content: ')' !important; }
+
+.FillVerticalSpace { grid-gap: 20px; margin-top: 5px; }
+
+td {
+	background-color: rgba(#fff, .5);
+	background-image:
+		linear-gradient(lightblue 2px, transparent 2px),
+		linear-gradient(90deg, lightblue 2px, transparent 2px),
+		linear-gradient(rgba(150, 150, 150, .3) 1px, transparent 1px),
+		linear-gradient(90deg, rgba(150, 150, 150, .3) 1px, transparent 1px);
+	background-size: 100px 100px, 100px 100px, 20px 20px, 20px 20px;
+	background-position: -3px 74px;
+	min-height: 1.5in;
+	max-width: 2in;
+}
+</style>
+++++
+
+For this page, you'll need to have @online-exercise{https://www.desmos.com/calculator/geyx4aetop, the Exploring Quadratics Desmos Activity} open on your computer. The parabola you see is the graph of @math{f(x)=x^2}. Another, *identical* parabola is hiding behind it. This second parabola is written in Vertex Form: @math{g(x)=a(x-h)^2+k}. Each coefficient starts at values to make @math{g(x)} equivalent to @math{f(x)}.
+
+@n Use the values of @math{a}, @math{h}, and @math{k} to write the Vertex Form of @math{x^2}: @fitb{}{@ifsoln{@math{1(x-0)^2+0}}}
+
```

Comment:
are you sure we want to ask about x^2 here and not f(x) or f(x) = x^2?

Reply:
good call